buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle

```diff
@@ -58,67 +63,165 @@ dependencies {
   testImplementation 'junit:junit:4.13.1'
 }
 
-public class ExampleRunnerTask extends JavaExec {
+class ExampleRunnerTask extends JavaExec {
 
-  @Input
-  public String basePackage = 'com.google.ads.googleads.examples.'
+  // The base package for all examples.
+  private String basePackage = 'com.google.ads.googleads.examples.'
```

Comment:
Will `runExample` still work from `google-ads-migration-examples` with these changes?

Reply:
Yes, it does. The example invocation in the documentation on devsite needs an update to the newest package name, will send a CL for that.

Reply:
Realized that these examples are not part of the printout - will rework a bit to include them.